migration/idempotent/001-base.sql

```diff
@@ -4,13 +4,22 @@
 CREATE OR REPLACE PROCEDURE _prom_catalog.execute_everywhere(command_key text, command TEXT, transactional BOOLEAN = true)
     SET search_path = pg_catalog
 AS $func$
+DECLARE
+    _is_restore_in_progress boolean = false;
 BEGIN
     IF command_key IS NOT NULL THEN
        INSERT INTO _prom_catalog.remote_commands(key, command, transactional) VALUES(command_key, command, transactional)
        ON CONFLICT (key) DO UPDATE SET command = excluded.command, transactional = excluded.transactional;
     END IF;
 
     EXECUTE command;
+
+    -- do not call distributed_exec if we are in the middle of restoring from backup
+    _is_restore_in_progress = coalesce((SELECT setting = 'on' from pg_catalog.pg_settings where name = 'timescaledb.restoring'), false);
```

Comment:
is cast to bool better?
`_is_restore_in_progress = coalesce((SELECT setting::boolean from pg_catalog.pg_settings where name = 'timescaledb.restoring'), false);`

Reply:
huh. I didn't realize 'on' would cast to true. wow